apollo-client/pom.xml

```diff
@@ -98,5 +98,10 @@
 			<scope>test</scope>
 		</dependency>
 		<!-- end of test -->
+		<dependency>
+			<groupId>io.kubernetes</groupId>
+			<artifactId>client-java</artifactId>
+			<version>18.0.0</version>
+		</dependency>
```

Comment:
1. Manage the version in the parent POM.
2. I'm considering marking the dependency as `optional`, as it's not required in every case, similar to the `spring` dependencies mentioned above.